x-pack/filebeat/module/misp/threat/config/pipeline.js

```diff
@@ -105,71 +114,87 @@ var threat = (function () {
                 if (arr.length == 2) {
                     filename = arr[0];
                     var sha256 = arr[1];
-                    attackPattern = '[' + 'file.hash.sha256 = ' + '\'' + sha256 + '\'' + ' AND file.path = ' + '\'' + filename + '\'' + ']';
+                    attackPattern = '[' + 'file:hash:sha256 = ' + '\'' + sha256 + '\'' + ' AND file:path = ' + '\'' + filename + '\'' + ']';
+                    attackPatternKQL = 'file.hash.sha256: ' + '"' + sha256 + '"' + ' AND file.path: ' + '"' + filename + '"';
                     evt.Put("file.hash.sha256", sha256);
                     evt.Put("file.path", filename);
                 }
                 break;
             case 'github-username':
-                attackPattern = '[' + 'github.username = ' + '\'' + v + '\'' + ']';
+                attackPattern = '[' + 'github:username = ' + '\'' + v + '\'' + ']';
+                attackPatternKQL = 'github.username: ' + '"' + v + '"';
```

Comment:
I would guess that in ECS we would populate the `user.name` with the user's github handle.

Reply:
OK, I will use that